internal/utils/timeutils/timeutils.go

```diff
@@ -0,0 +1,20 @@
+package timeutils
+
+import (
+	"time"
+)
+
+// FormalLocal takes a timestamp in string format (inputLayout) and converts it into a readable
+// format in the local timezome (outputLayout).
+func FormatLocal(timeString string) string {
+	inputLayout := "2006-01-02T15:04:05.999999Z07:00"
+	outputLayout := "2 January 2006 3:04:05 PM PST"
+
+	timestamp, err := time.Parse(inputLayout, timeString)
+	if err != nil {
+		return timeString
+	}
+
+	timestampDefaultTZ := timestamp.Local()
+	return timestampDefaultTZ.Format(outputLayout)
+}
```

Comment:
The input layout should use `time.RFC3339Nano` constant instead. Also this should be parsed on the caller side. Is there a way to make the graphql library parse the time.Time by defining a literal type or something?

Reply:
the library doesnt parse datetime :( they have an open issue, i don't think it's really being worked on https://github.com/shurcooL/githubv4/issues/9

Reply:
Looks like it works https://github.com/shurcooL/githubv4#scalar-types